gitlab.go

```diff
@@ -524,6 +525,10 @@ func rateLimitBackoff(min, max time.Duration, _ int, resp *http.Response) time.D
 					min = wait
 				}
 			}
+		} else {
+			// For each attempt without the header, back off an additiona 100% exponentially. With the default milliseconds
+			// being set to 100 for `min`, this makes the 5th retry wait 3.2 seconds (3,200 ms) by default
```

Comment:
Please use the suggested as it 1) fixes a typo 2) adds a missing period 3) keeps the comment length below 80 chars:

```suggestion
			// In case the RateLimit-Reset header is not set, back off an additional
			// 100% exponentially. With the default milliseconds being set to 100 for
			// `min`, this makes the 5th retry wait 3.2 seconds (3,200 ms) by default.
```